app/src/main/java/io/apicurio/registry/mt/TenantRequestFilter.java

```diff
@@ -17,34 +17,80 @@
 package io.apicurio.registry.mt;
 
 import io.apicurio.registry.rest.Headers;
+import io.quarkus.runtime.StartupEvent;
+import io.quarkus.vertx.web.RouteFilter;
+import io.vertx.ext.web.RoutingContext;
 
+import javax.enterprise.event.Observes;
 import javax.inject.Inject;
-import javax.ws.rs.container.ContainerRequestContext;
-import javax.ws.rs.container.ContainerRequestFilter;
-import javax.ws.rs.ext.Provider;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import java.io.IOException;
 
 /**
  * @author eric.wittmann@gmail.com
  */
-@Provider
-public class TenantRequestFilter implements ContainerRequestFilter {
+public class TenantRequestFilter {
```

Comment:
Am I missing something or you forgot to annotate this class with something else?

Reply:
It's enough with the vertx annotation